src/views/Dashboard/View.vue

```diff
@@ -71,6 +71,24 @@ export default {
       user: this.$store.getters.getUser.Username
     }
   },
+  methods:{
+    delmail() {
+      this.isLoading = true;
+      fetch("https://speedwagonmailback.herokuapp.com/email/del", {
+        method: "POST",
+        body: JSON.stringify({
+          id: this.mail._id
+        }),
+        headers: {
+          "content-type": "application/json"
+        }
+      }).then(response => response.json()
+      ).then(result => {
+        this.$router.push("/dashboard")
```

Comment:
Disini tambahin juga we sebelum diredirect, 
Kalo deleteberhasil, delete localstorage.Mailnya dulu juga we